### ISSUE TITLE
Cannot stop chat app

### ISSUE THREAD

**Issue by TechInterMezzo** [flomesh-io/ztm#49]
I cannot stop the chat app by command line or web interface. I get now error when I do that. But either the stopping does not work or it gets restarted right away.

**Comment by pajama-coder** [comment]
Yes, to stop an app completely is kinda tricky right now. Currently you'll have to restart the agent after stopping an app to make it really stop running. But the app will start automatically next time you use it.

We'll look to add a "disable app" function so it won't auto-start, if that meets your need.

@TechInterMezzo
